### PR TITLE
Passing sgRNA sequences to regular and Batch D3 plots

### PR DESCRIPTION
**Pull request by trevormartinj7**
https://linear.app/edilytics/issue/EDI-17/d3-improvements

### CODE REVIEW

**Review by Colelyman — APPROVED** [review]
This works and looks good to me. Let's merge it in after we merge in the corresponding branch in the tests repo.

**Comment by Colelyman** [comment]
@trevormartinj7 After we merge in the corresponding test branch (and point the Github Actions integration_tests.yml back to master), I think this will be ready to merge in. Is there anything else that you think needs to be changed?

**Comment by trevormartinj7** [comment]
> @trevormartinj7 After we merge in the corresponding test branch (and point the Github Actions integration_tests.yml back to master), I think this will be ready to merge in. Is there anything else that you think needs to be changed?

Yeah, once we swap the test branch I think it looks good! I ran a large_batch test with this new branch and it seems to be behaving exactly as expected. Approving now!

**Pull request merged**
